cubie-a7*: dts: rename soundcard-mach,name

CodeChenL opened the pull request:
To avoid confusion with other devices when matching ALSA UCM configurations Link: https://github.com/radxa-pkg/radxa-alsa-config/pull/13

CodeChenL committed:
cubie-a7a: dts: rename ac101 soundcard-mach,name

To avoid confusion with other devices when matching ALSA UCM configurations
Link: https://github.com/radxa-pkg/radxa-alsa-config/pull/13

Signed-off-by: Jiali Chen <chenjiali@radxa.com>

RadxaYuntian reviewed (commented):
这里好多disabled的节点，之后可以清理一下

CodeChenL committed:
cubie-a7*: dts: rename hdmi soundcard-mach,name

To avoid confusion with other devices when matching ALSA UCM configurations
Link: https://github.com/radxa-pkg/radxa-alsa-config/pull/13

Signed-off-by: Jiali Chen <chenjiali@radxa.com>

RadxaYuntian reviewed (changes requested):
整理提交

RadxaStephen commented:
Merged.
